manifests/init.pp

```diff
@@ -316,6 +323,7 @@
   ~> class { '::foreman::install': }
   ~> class { '::foreman::config': }
   ~> class { '::foreman::database': }
+  ~> class { '::foreman::tasks_service': }
```

Comment:
I can certainly turn the include in `plugin/tasks.pp` into a `contain ::foreman::service::jobs`. However here I'm supposed to make it start (and return OK) before starting Foreman itself. Do you want me to move this class declaration to contain too (if so, how?)?